SA3/qa_web/templates/qa_web/home.html

```diff
@@ -7,7 +7,7 @@
       <div class="container">
         <h1 class="masthead-heading mb-0">Got Questions</h1>
         <h2 class="masthead-subheading mb-0"> ? !</h2>
-        <a {% if user.is_authenticated %} href="/questions/" {% else %}  href="/login/"  {% endif %} class="btn btn-primary btn-xl rounded-pill mt-5">Click to Ask a Question</a>
+        <a href="/questions/" class="btn btn-primary btn-xl rounded-pill mt-5">Click to Ask a Question</a>
```

Comment:
Letting the backend handle the redirecting and authentication is a good way to go about it.